kentik_api_library/tests/e2e/cloudexport/test_ibm.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+import pytest
+from numpy import isin
```

Comment:
Why this? I don't think it is used anywhere (which is really good).

Reply:
Sneaked in here unnoticed, removing